fe/src/main/java/org/apache/doris/load/routineload/KafkaRoutineLoadJob.java

```diff
@@ -62,19 +63,17 @@
 public class KafkaRoutineLoadJob extends RoutineLoadJob {
     private static final Logger LOG = LogManager.getLogger(KafkaRoutineLoadJob.class);
 
-    private static final int FETCH_PARTITIONS_TIMEOUT_SECOND = 5;
+    public static final String KAFKA_FILE_CATALOG = "kafka";
```

Comment:
The catalog is a static param of job while the catalog of file is created by user.

Reply:
Yes, in my design , catalog named 'kafka' is a reserved keyword, which is exactly used by kafka client.